[2025-09-21T11:31:06.060+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:30:00+00:00/task_id=trigger_model_training permission to 509
[2025-09-21T11:31:06.060+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:30:00+00:00/task_id=trigger_model_training permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:30:00+00:00/task_id=trigger_model_training'
[2025-09-21T11:31:06.103+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:30:00+00:00/task_id=trigger_model_training permission to 509
[2025-09-21T11:31:06.105+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:30:00+00:00/task_id=trigger_model_training permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:30:00+00:00/task_id=trigger_model_training'
[2025-09-21T11:31:06.156+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.trigger_model_training scheduled__2025-09-21T11:30:00+00:00 [queued]>
[2025-09-21T11:31:06.169+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.trigger_model_training scheduled__2025-09-21T11:30:00+00:00 [queued]>
[2025-09-21T11:31:06.169+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-09-21T11:31:06.190+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): trigger_model_training> on 2025-09-21 11:30:00+00:00
[2025-09-21T11:31:06.201+0000] {standard_task_runner.py:57} INFO - Started process 1003 to run task
[2025-09-21T11:31:06.205+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'trigger_model_training', 'scheduled__2025-09-21T11:30:00+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmpwtup0krx']
[2025-09-21T11:31:06.209+0000] {standard_task_runner.py:85} INFO - Job 83: Subtask trigger_model_training
[2025-09-21T11:31:06.281+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:30:00+00:00/task_id=trigger_model_training permission to 509
[2025-09-21T11:31:06.282+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:30:00+00:00/task_id=trigger_model_training permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:30:00+00:00/task_id=trigger_model_training'
[2025-09-21T11:31:06.284+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.trigger_model_training scheduled__2025-09-21T11:30:00+00:00 [running]> on host 36da5371e73d
[2025-09-21T11:31:06.395+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='trigger_model_training' AIRFLOW_CTX_EXECUTION_DATE='2025-09-21T11:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-21T11:30:00+00:00'
[2025-09-21T11:31:11.307+0000] {python.py:194} INFO - Done. Returned value was: c59e1660ceb648a5a32d3a5c4e564988
[2025-09-21T11:31:11.382+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=trigger_model_training, execution_date=20250921T113000, start_date=20250921T113106, end_date=20250921T113111
[2025-09-21T11:31:11.454+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-09-21T11:31:11.482+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
